src/main/java/net/minestom/server/instance/Chunk.java

```diff
@@ -227,8 +160,15 @@ public int getMaxSection() {
      *
      * @return true if this chunk is affected by a {@link Generator}
      */
-    public boolean shouldGenerate() {
-        return shouldGenerate;
+    boolean shouldGenerate();
+
+    /**
+     * Gets the world position of this chunk.
+     *
+     * @return the position of this chunk
+     */
+    default Point toPosition() {
```

Comment:
toPosition seems like a weird name, its just getting the world position of the chunk not really doing any conversion per say as "to" implies